eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyFilesystemSources.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2023 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven.objectionary;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.cactoos.Input;
+import org.cactoos.io.InputOf;
+
+/**
+ * Objectionary stored locally in the filesystem.
+ * The aim of this class is to download object sources directly from the filesystem, for example,
+ * from the project itself. It is useful for testing purposes.
+ *
+ * @since 0.30
+ * @todo #2125:30min Add unit tests for OyFilesystemSources. The tests should check that
+ *  OyFilesystemSources can get an object from the filesystem and that it can check if an object
+ *  exists in the filesystem.
+ */
+public final class OyFilesystemSources implements Objectionary {
```

Comment:
@volodya-lombrozo Does it really imply multiple sources per instance? Maybe `OyFilesystem` is better?